src/ios/AKPlugin.m

```diff
@@ -60,8 +60,19 @@ - (void)loginWithEmail:(CDVInvokedUrlCommand *)command {
 }
 
 - (void)getAccount:(CDVInvokedUrlCommand *)command {
+  if (_accountKit == nil) {
+      _accountKit = [[AKFAccountKit alloc] initWithResponseType:AKFResponseTypeAccessToken];
```

Comment:
We want to check if the user has already logged in once in the app. If you always return an error when _accountKit is not initialised then you will force the user to log in once more with SMS even though the tokens are cached and present.

At least this is my experience when testing with the emulator, stopping and starting the app.